src/features/resources/StepComponent.tsx

```diff
@@ -0,0 +1,462 @@
+                align="center"
+                showLeftLine={true}
+                color="#DA9694"
+                textWidth="150px"
+                gradientType="second"
+              />
+              <ResourcesHeading variant="h2">
+                {currentStep?.stepName}
+              </ResourcesHeading>
+              <Box
+                sx={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                }}
+              ></Box>
+              <Box
+                sx={{
+                  mt: '30px',
+                  display: { xs: 'none', sm: 'flex' },
+                  alignItems: 'center',
+                }}
+              >
+                <Breadcrumbs
+                  aria-label="breadcrumb"
+                  separator={
+                    <Image
+                      src={'/icons/right-arrow.svg'}
+                      alt={'icon'}
+                      width={9}
+                      height={9}
+                      style={{ transform: 'translateY(20%)' }}
+                    />
+                  }
+                >
+                  {breadcrumbs.map((title, index) => (
+                    <BreadcrumbsHeading key={index} variant="body2">
+                      {title}
+                    </BreadcrumbsHeading>
+                  ))}
+                </Breadcrumbs>
+              </Box>
+              <Box sx={{ my: { xs: '20px', sm: '30px' } }}>
+                <ResourcesSubHeading>
+                  {breadcrumbs.length > 0 && (
+                    <span style={{ fontWeight: 'bold', marginTop: '8px' }}>
+                      {breadcrumbs[breadcrumbs.length - 1]}
+                    </span>
+                  )}
+                </ResourcesSubHeading>
+                <ResourcesPara variant="body1" sx={{ textAlign: 'start' }}>
+                  {currentStep?.stepHeading}
+                </ResourcesPara>
+              </Box>
+              <Box>
+                {currentStep?.stepNumber === 5 ? (
+                  <ResourcesTableContainer>
+                    <StyledTable sx={{ minWidth: 768 }}>
+                      <TableHead>
+                        <TableRow>
+                          {Array.isArray(options) &&
+                            options.map((option, index) =>
+                              option?.headings?.map((heading, headingIndex) => (
+                                <TableCell
+                                  key={`${index}-${headingIndex}`}
+                                  sx={{
+                                    fontWeight: 'bold',
+                                    '&:nth-last-child(1)': {
+                                      textAlign: 'right',
+                                    },
+                                  }}
+                                >
+                                  {heading}
+                                </TableCell>
+                              ))
```

Comment:
As stated above, this should be a separate component. I have shared an example on slack from the fast-clean project, please refer to that and/or reach out if you need more clarity on this implementation

Reply:
```

const BookingForm = () => {
  const { currentStep, formData, nextStep } = useMultiStepForm();

  console.log("BookingForm - Current Step:", currentStep);

  if (currentStep === 1) return <LocationSelection />;
  else if (currentStep === 2) return <LiscencePlateStep />;
  else if (currentStep === 3) return <CarTypeStep />;
  else if (currentStep === 4) return <PackageSelectionStep />;
  else if (currentStep === 5) return <SubscriptionPackagesStep />;
  else if (currentStep === 6 && formData.selectedPackageType === "Subscription Plans") {
    // nextStep();
    return null;
  } else if (currentStep === 6 && formData.selectedPackageType === "Anywhere Autocare") return <AutocarePackagesStep />;
  else if (currentStep === 7) return <AdditionalOptionsStep />;
  else if (currentStep === 8) return <DetailingStep />;
  else if (currentStep === 9) return <SelectCityStep />;
  else if (currentStep === 10) return <ScheduleAppointmentStep />;
  else if (currentStep === 11) return <SummaryStep />;
  else if (currentStep === 12) return <PersonParticularsStep />;

  return <Box>Undefined</Box>;
};

export default BookingForm;
```

Here is some sample code from the Fast Clean project, I'd like you to focus on how all the different components are rendered according to their step number